tests/settings.py

```diff
@@ -109,6 +109,29 @@
     },
 ]
 
+# This setting is used by the "test_auth_fail_template_tag" test
+BAD_TEMPLATES_SETTING = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(PROJECT_PATH, 'tests/templates'),
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.contrib.messages.context_processors.messages',
+            ],
+            'debug': True,
+        },
+    },
+]
+
```

Comment:
We do have to set a new dictionary  with override_settings, but I think we could do something a little more DRY here. How about copying the original templates dictionary here and then replacing just the context_processors member in the copy?